webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/AdHocMeshingService.scala

```diff
@@ -23,69 +23,69 @@ import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext}
 import scala.reflect.ClassTag
 
-case class IsosurfaceRequest(dataSource: Option[DataSource],
-                             dataLayer: SegmentationLayer,
-                             cuboid: Cuboid,
-                             segmentId: Long,
-                             subsamplingStrides: Vec3Int,
-                             scale: Vec3Double,
-                             mapping: Option[String] = None,
-                             mappingType: Option[String] = None,
-                             findNeighbors: Boolean = true)
+case class AdHocMeshRequest(dataSource: Option[DataSource],
+                            dataLayer: SegmentationLayer,
+                            cuboid: Cuboid,
+                            segmentId: Long,
+                            subsamplingStrides: Vec3Int,
+                            scale: Vec3Double,
+                            mapping: Option[String] = None,
+                            mappingType: Option[String] = None,
+                            findNeighbors: Boolean = true)
 
 case class DataTypeFunctors[T, B](
     getTypedBufferFn: ByteBuffer => B,
     copyDataFn: (B, Array[T]) => Unit,
     fromLong: Long => T
 )
 
-class IsosurfaceActor(val service: IsosurfaceService, val timeout: FiniteDuration) extends Actor {
+class AdHocMeshingActor(val service: AdHocMeshingService, val timeout: FiniteDuration) extends Actor {
 
   def receive: Receive = {
-    case request: IsosurfaceRequest =>
-      sender() ! Await.result(service.requestIsosurface(request).futureBox, timeout)
+    case request: AdHocMeshRequest =>
+      sender() ! Await.result(service.requestAdHocMesh(request).futureBox, timeout)
     case _ =>
-      sender() ! Failure("Unexpected message sent to IsosurfaceActor.")
+      sender() ! Failure("Unexpected message sent to AdHocMeshingActor.")
   }
 }
 
-class IsosurfaceService(binaryDataService: BinaryDataService,
-                        mappingService: MappingService,
-                        actorSystem: ActorSystem,
-                        isosurfaceTimeout: FiniteDuration,
-                        isosurfaceActorPoolSize: Int)(implicit ec: ExecutionContext)
+class AdHocMeshingService(binaryDataService: BinaryDataService,
```

Comment:
```suggestion
class AdHocMeshService(binaryDataService: BinaryDataService,
```

I think I’d prefer it without the “ing” since that would be the only place where the service would be named with a conjucated form.

Reply:
I removed all ings